decidim-core/app/assets/stylesheets/decidim/modules/_omnipresent_banner.scss

```diff
@@ -0,0 +1,20 @@
+.omnipresent-banner{
+  padding: .5em 1em;
+  text-align: center;
+  background-color: #ff414d;
```

Comment:
Should we use the main organization's color to follow the color scheme more closely?

Reply:
@htmlboy should the omnipresent banner change its collor according to main organization's color?

Reply:
Yes, this is how I prepared it in this PR: https://github.com/decidim/design/pull/178

(see: _top-banner.scss)